app/assets/stylesheets/utils/_variables.scss

```diff
@@ -23,3 +24,6 @@ $text-black-border: $black 1px 0px 0px, $black 0.540302px 0.841471px 0px,
   $black -0.416147px 0.909297px 0px, $black -0.989992px 0.14112px 0px,
   $black -0.653644px -0.756802px 0px, $black 0.283662px -0.958924px 0px,
   $black 0.96017px -0.279415px 0px;
+
+// Font Family
+$font: "Roboto", sans-serif;
```

Comment:
We will be using different fonts in this project and this name could lead to confusion, you can change "font" to primaryFont, titleFont, etc.

Reply:
done! :)